conda/base/context.py

```diff
@@ -1254,6 +1254,34 @@ def auto_activate_base(self) -> bool:
     def default_activation_env(self) -> str:
         return self._default_activation_env or ROOT_ENV_NAME
 
+    @property
+    def environment_context_keys(self) -> list[str]:
```

Comment:
I don't think this needs to be it's own function here per se. But, trying to follow the pattern used other places in this module, for example `category_map`.

Reply:
Hm, I kind of agree with you, but the alternative (place it in `Environment`) is not better either. I'll defer to others for a final say, but this is not a blocker imo.